src/compiler/watch.ts

```diff
@@ -521,6 +528,22 @@ namespace ts {
         ExtendedConfigOfReferencedProject: "Extended config file of referenced project",
         WildcardDirectoryOfReferencedProject: "Wild card directory of referenced project",
         PackageJson: "package.json file",
+
+        // Additional tsserver specific watch information
+        /* @internal */

```

Comment:
Everything in this file is already internal ?

Reply:
So it is, oops. I could have sworn it wasn't on my branch (maybe an error in my transformer code).

Now I'm even more compelled to make this an enum.